web/src/main/java/co/mobileaction/example/web/controller/PostController.java

```diff
@@ -41,4 +42,26 @@ public ResponseEntity<Boolean> deletePost(@PathVariable Long postId)
 
         return ResponseEntity.ok(true);
     }
+
+    @ResponseStatus(HttpStatus.CREATED)
+    @PostMapping
+    public void createPost(@RequestBody Post post)
+    {
+        postService.savePost(post);
+    }
+
+    @DeleteMapping("/user/{userId}")
+    public ResponseEntity<Boolean> deletePostsOfUser(@PathVariable Long userId)
+    {
+        List<Post> userPosts = postService.findAllPostsOfUser(userId);
+        postService.deleteUserPosts(userPosts);
+
+        return ResponseEntity.ok(true);
+    }
+
+    @GetMapping("/user")
+    public List<Long> getDistinctUsersFromPosts()
+    {
+        return postService.findDistinctUsersFromPosts();
+    }

```

Comment:
I think this endpoint is not in the requirement but I guess you wanted to check distinct userIds via an endpoint.